### PR TITLE
update backend

### PR DESCRIPTION
**Pull request by malachitech**
I just created a server for the project.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//             }
+//             console.log('server is running'.bgCyan)
+//        })
+//        console.log(process.env.PORT)
+
+//        console.log(`Database is up and running on ${conn.connection.host}`.bgGreen.underline)
+//     }catch(err){
+//       console.log(`${err}`.bgRed.underline)
+//     }
+// }
```
Comment (Olaoluwa402) [inline backend/server.js:55]: Remove this code

**Review by ABIKEM01 — APPROVED** [review]
I have reviewed  and its okay by me

File: backend/server.js (hunk 1 above)
Comment (mfontosco) [reply backend/server.js:55]: Good

**Review by Olaoluwa402 — APPROVED** [review]
Effect the reviewed comments

File: backend/server.js (hunk 1 above)
Comment (malachitech) [reply backend/server.js:55]: ok sir

**Pull request merged**